libs/core/src/lib/switch/switch.component.html

```diff
@@ -7,8 +7,8 @@
     [class.is-disabled]="disabled"
     [attr.aria-label]="ariaLabel"
 >
-    <span *ngIf="activeText || inactiveText" class="fd-switch__text">
-        {{ isChecked ? activeText : inactiveText }}
+    <span *ngIf="(isChecked ? activeText : inactiveText) as text" class="fd-switch__text">
```

Comment:
I am a little confused by this check, why should we add `as text` expression?
May be this checking would be more understandable `!!(isChecked ? activeText : inactiveText)`

Reply:
@Smiranin I'm just avoiding keeping the same condition in 2 places: in the `ngIf` and below as an interpolation.

If you want, I can make a getter on the component level instead. But in my opinion it's better to not pollute component with extra properties without a need.

Reply:
Ok. No need